dist/api_data.py

```diff
@@ -1025,6 +1025,9 @@ def __ge__(self, other):
     behavior of WiredTiger when live restoring from a backup.''', type='category', subconfig = [
         Config('enabled', 'false', r'''whether live restore is enabled or not.''', type='boolean'),
         Config('path', '', r'''the path to the backup that will be restored from.'''),
+        Config('read_size', '1MB', r'''
+            the read size for data migration. This setting is a best effort. It does not force every
+            read to be this size.''', min='512B', max='16MB'),
```

Comment:
Said this in the other thread, but this now needs to document that it must be power of 2. See 'allocation_size' for some text to copy.

Reply:
042d20ceb